### PR TITLE
MAUI-1844-Upcoming features list added.

### PR DESCRIPTION
**Pull request by MuniappanSubramanian**
Hi @Jeyasri-Murugan

I have prepared UG for migrating MAUI SfScheduler from Xamarin SfSchedule. Please review the changes and let me know if you have any concerns.

Covered Below Topics:

- Assembly Name
- Namespaces
- Properties
- Events
- Enums
- Classes
- Methods
- SchedulerAppointment
- SchedulerAppointmentMapping
- MonthView
- DayView
- TimelineView
- SchedulerTimeRegion
- AppointmentTextStyle
- CellStyle
- HeaderView
- ViewHeaderSettings
- WeekNumberStyle

Regards,
Muniappan S

### CODE REVIEW

File: MAUI/Scheduler/migration.md [hunk 1]
```diff
@@ -301,6 +342,56 @@ To migrate easier from [Xamarin SfSchedule](https://www.syncfusion.com/xamarin-u
...
 <td>Gets or sets the properties which allows to customize all the properties of view header in the scheduler.</td></tr>
 
+<tr>
+<td>{{'[AppointmentViewLayout](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.SfSchedule.html#Syncfusion_SfSchedule_XForms_SfSchedule_AppointmentViewLayout)' | markdownify }}</td>
+<td>{{'[AppointmentTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerTimeSlotView.html?tabs=tabid-9%2Ctabid-3%2Ctabid-5%2Ctabid-7%2Ctabid-1#Syncfusion_Maui_Scheduler_SchedulerTimeSlotView_AppointmentTemplate)'| markdownify }}</td>
+<td>Gets or sets the appointment template to customize the default UI.</td></tr>
+
+<tr>
+<td>{{'[VerticalLineColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.SfSchedule.html#Syncfusion_SfSchedule_XForms_SfSchedule_TimeZone)' | markdownify }}</td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:351]: help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.SfSchedule.html#Syncfusion_SfSchedule_XForms_SfSchedule_TimeZone -  update vertcial line color property

File: MAUI/Scheduler/migration.md [hunk 2]
```diff
@@ -161,12 +161,53 @@ To migrate easier from [Xamarin SfSchedule](https://www.syncfusion.com/xamarin-u
...
+<td>Gets or sets the appointment template to customize the default UI.</td></tr>
+
+<tr>
+<td>{{'[MonthInlineAppointmentTappedEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.MonthInlineAppointmentTappedEventArgs.html)'| markdownify }}</td>
+<td>{{'[SchedulerTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerTappedEventArgs.html)'| markdownify }}</td>
+<td>Occurs when the user clicks or touch on the scheduler elements.</td></tr>
+
+<tr>
+<td>{{'[MonthInlineLoadedEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.MonthInlineLoadedEventArgs.html)'| markdownify }}</td>
+<td>{{'[AppointmentTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerTimeSlotView.html?tabs=tabid-9%2Ctabid-3%2Ctabid-5%2Ctabid-7%2Ctabid-1#Syncfusion_Maui_Scheduler_SchedulerTimeSlotView_AppointmentTemplate)'| markdownify }}</td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:188]: Add SchedulerMonthView.AppointmentTemplate for month view like in needed places mention SchedulerDayView.AppointmentTemplate , SchedulerTimelineView.AppointmentTemplate link

File: MAUI/Scheduler/migration.md [hunk 3]
```diff
@@ -161,12 +161,53 @@ To migrate easier from [Xamarin SfSchedule](https://www.syncfusion.com/xamarin-u
...
-<td>Represents a class which is used to configure all the properties of Month view..</td></tr>
+<td>Represents a class which is used to configure all the properties of Month view.</td></tr>
 
 <tr>
 <td>{{'[TimelineLabelSettings](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.TimelineLabelSettings.html)'| markdownify }}</td>
 <td>{{'[SchedulerTimelineView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerTimelineView.html?tabs=tabid-3%2Ctabid-5)'| markdownify }}</td>
 <td>Represents a class which is used to configure all the properties of timeline day, timeline week, timeline workweek and timeline month views of the SfScheduler.</td></tr>
+
+<tr>
+<td>{{'[AppointmentLoadedEventArgs](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.AppointmentLoadedEventArgs.html)'| markdownify }}</td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:172]: mention link SchedulerMonthView.AppointmentTemplate , SchedulerDayView.AppointmentTemplate , SchedulerTimelineView.AppointmentTemplate link

File: MAUI/Scheduler/migration.md [hunk 4]
```diff
@@ -391,6 +482,16 @@ To migrate easier from [Xamarin SfSchedule](https://www.syncfusion.com/xamarin-u
 <td>{{'[Location](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerAppointment.html#Syncfusion_Maui_Scheduler_SchedulerAppointment_Location)'| markdownify }}</td>
 <td>Gets or sets the Location value for Scheduler Appointment to describe the location of the appointment in scheduler.</td></tr>
 
+<tr>
+<td>{{'[MinHeight](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.ScheduleAppointment.html#Syncfusion_SfSchedule_XForms_ScheduleAppointment_MinHeight)'| markdownify }}</td>
+<td>{{'[MinimumAppointmentDuration](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerTimeSlotView.html?tabs=tabid-9%2Ctabid-3%2Ctabid-5%2Ctabid-7%2Ctabid-1#Syncfusion_Maui_Scheduler_SchedulerTimeSlotView_MinimumAppointmentDuration)'| markdownify }}</td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:487]: mention SchedulerDayView.MinimumAppointmentDuration, SchedulerTimelineView.MinimumAppointmentDuration link

File: MAUI/Scheduler/migration.md [hunk 5]
```diff
@@ -466,6 +567,16 @@ To migrate easier from [Xamarin SfSchedule](https://www.syncfusion.com/xamarin-u
 <td>{{'[Location](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerAppointmentMapping.html#Syncfusion_Maui_Scheduler_SchedulerAppointmentMapping_Location)'| markdownify }}</td>
 <td>Gets or sets the Location Mapping to map the custom property with scheduler appointments location property.</td></tr>
 
+<tr>
+<td>{{'[MinHeightMapping](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.ScheduleAppointmentMapping.html#Syncfusion_SfSchedule_XForms_ScheduleAppointmentMapping_MinHeightMapping)'| markdownify }}</td>
+<td>{{'[MinimumAppointmentDuration](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Scheduler.SchedulerTimeSlotView.html?tabs=tabid-9%2Ctabid-3%2Ctabid-5%2Ctabid-7%2Ctabid-1#Syncfusion_Maui_Scheduler_SchedulerTimeSlotView_MinimumAppointmentDuration)'| markdownify }}</td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:572]: mention SchedulerDayView.MinimumAppointmentDuration, SchedulerTimelineView.MinimumAppointmentDuration link

Comment (MuniappanSubramanian) [reply MAUI/Scheduler/migration.md]: @Jeyasri-Murugan As per your concerns, Properties updated.

File: MAUI/Scheduler/migration.md (hunk 4 above)
Comment (MuniappanSubramanian) [reply MAUI/Scheduler/migration.md]: @Jeyasri-Murugan As per your concerns, Properties updated.

File: MAUI/Scheduler/migration.md (hunk 3 above)
Comment (MuniappanSubramanian) [reply MAUI/Scheduler/migration.md:172]: @Jeyasri-Murugan As per your concerns, Properties updated .

File: MAUI/Scheduler/migration.md (hunk 2 above)
Comment (MuniappanSubramanian) [reply MAUI/Scheduler/migration.md]: @Jeyasri-Murugan As per your concerns, Properties updated .

File: MAUI/Scheduler/migration.md (hunk 1 above)
Comment (MuniappanSubramanian) [reply MAUI/Scheduler/migration.md]: @Jeyasri-Murugan As per your concerns, API link updated.

**Comment by MuniappanSubramanian** [comment]
Rebuild

**Review by Jeyasri-Murugan — APPROVED** [review]

**Comment by MuniappanSubramanian** [comment]
rebuild

**Pull request merged**